examples/hotrod/docker-compose.yml

```diff
@@ -6,6 +6,7 @@ services:
       - "16686:16686"
       - "4318:4318"
     environment:
+      - COLLECTOR_OTLP_ENABLED=true
```

Comment:
this is no longer needed in the latest versions. Check which `latest` your Docker has.
```
$ docker images | grep all-in-one
```

Reply:
Oops, sorry. You're right! My `latest` version of all-in-one image is not updated. I've updated the latest version and re-checked now it's working without `COLLECTOR_OTLP_ENABLED` env... This issue seems not trivial then. I'll close this PR.